libsql-server/src/connection/libsql.rs

```diff
@@ -391,14 +391,43 @@ where
         ctx: RequestContext,
         builder: B,
     ) -> Result<(B, Program)> {
+        struct Bomb {
+            canceled: Arc<AtomicBool>,
+            defused: bool,
+        }
+
+        impl Drop for Bomb {
+            fn drop(&mut self) {
+                if !self.defused {
+                    tracing::debug!("cancelling request");
```

Comment:
can we improve botht he debug logs so they are unique and easier to follow. I think seeing a bunch of these in the logs when turning on debug will be annoying so mayb we consider going to trace? maybe for atleast one of them?

Reply:
Hopefully not too many request should get cancelled, but no worries dowgrading to trace